### PR TITLE
Update references to CNCF landscape (v2)

### LINKED ISSUE #44022: Adopt v2 Landscape
**This is a Feature Request**

**What would you like to be added**
- Prepare to adopt https://cncf.landscape2.io/
- Actually adopt it

**Why is this needed**
https://cncf.landscape2.io/ will one day supersede https://landscape.cncf.io/guide

We use the CNCF landscape on pages such as https://kubernetes.io/training/

**Comments**
/area web-development

### PR DESCRIPTION
**Pull request by cynthia-sg**
This PR updates all the references to the CNCF landscape (v2). This includes migrating to the new embeddable views that the new landscape provides, changing how iframe-resizer is used and updating a few links.

Related to #44022

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
@cynthia-sg I see the bot has added `do-not-merge/invalid-commit-message` as the commit message has closes#<issue number> ([Blocked Keywords Guide](https://www.kubernetes.dev/docs/guide/pull-requests/#do-not-use-github-keywords-or-mentions-within-your-commit-message)). Would you be able to edit the commit message in order to move this PR forward?

[Message from bot:](https://github.com/kubernetes/website/pull/44792#issuecomment-1898142353)

> [Keywords](https://help.github.com/articles/closing-issues-using-keywords) which can automatically close issues and at(@) or hashtag(#) mentions are not allowed in commit messages.
> 
> The list of commits with invalid commit messages:
> 
> [ebfd1cd](https://github.com/kubernetes/website/commits/ebfd1cda0e7d4846c7f37bb0cbb27f59c34525db) Update references to CNCF landscape (v2)

**Comment by cynthia-sg** [comment]
Done 👍

**Review by nate-double-u — COMMENTED** [review]
Thanks for this @cynthia-sg!

Normally we don't update all localizations together like this, but as this is a very thorough one time update relating to an outside change, I wonder if it's OK as is? Same with old blog articles, we normally don't edit entries older than one year, but given all the work has been done, I'd like to merge this in one go if we can.

Deploy previews:
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/training/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/partners/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/case-studies/netease/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/docs/setup/production-environment/turnkey-solutions/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/docs/tasks/debug/debug-cluster/resource-usage-monitoring/

Blog
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/blog/2020/12/02/dockershim-faq/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/blog/2021/11/12/are-you-ready-for-dockershim-removal/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/blog/2022/02/17/dockershim-faq/

localizations
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/de/partners/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/de/training/

—

* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/es/partners/

—

* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/fr/partners/

—

* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/hi/docs/setup/production-environment/turnkey-solutions/

—

* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/it/partners/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/it/training/

—

* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/ja/blog/2022/02/17/dockershim-faq/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/ja/docs/setup/production-environment/turnkey-solutions/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/ja/partners/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/ja/training/

—

* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/ko/docs/setup/production-environment/turnkey-solutions/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/ko/partners/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/ko/training/

—

* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/pl/training/


—

* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/pt-br/blog/2022/02/17/dockershim-faq/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/pt-br/partners/

—

* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/vi/partners/

—

* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/zh-cn/blog/2020/12/02/dockershim-faq/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/zh-cn/blog/2022/02/17/dockershim-faq/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/zh-cn/case-studies/netease/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/zh-cn/docs/setup/production-environment/turnkey-solutions/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tasks/debug/debug-cluster/resource-usage-monitoring/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/zh-cn/partners/
* https://deploy-preview-44792--kubernetes-io-main-staging.netlify.app/zh-cn/training/

/lgtm

File: layouts/shortcodes/cncf-landscape.html [hunk 1]
```diff
@@ -57,11 +57,27 @@
...
+    iFrameResize({ }, '#iframe-landscape');
+  </script>
   {{ else }}
-  <iframe frameborder="0" id="landscape" scrolling="no" src="https://landscape.cncf.io/card-mode?embed=yes" style="width: 1px; min-width: 100%" title="CNCF Landscape"></iframe>
+  <style>
+    /* Display subcategory name under sticky header when using anchor */
+    #iframe-landscape-kcsp, #iframe-landscape-conformance, #iframe-landscape-ktp {
+      scroll-margin-top: 35px;
+    }
+  </style>
```
Comment (sftim) [inline layouts/shortcodes/cncf-landscape.html:72]: We'd prefer to have this in the page header (use `.HasShortcode` to handle that).

**Review by sftim — APPROVED** [review]
Thanks. A nit, but let's merge it.

/area web-development
/approve

**Comment by sftim** [comment]
I think it is OK to change each localization in order to land this fix.

File: layouts/shortcodes/cncf-landscape.html [hunk 2]
```diff
@@ -57,11 +57,27 @@
 </script>
 {{- end -}}
 <div id="frameHolder">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.9/iframeResizer.min.js"></script>
```
Comment (cjyabraham) [inline layouts/shortcodes/cncf-landscape.html:60]: You may want to bring this script local to this repo so you don't have this external dependency.

**Pull request merged**